bin/merge.sh

```diff
@@ -19,18 +19,15 @@
 # limitations under the License.
```

Comment:
Before merge, update https://github.com/openhwgroup/core-v-verif/issues/2134

(Let's leave this comment up until right before merge.)